playbooks/openstack/configuration.md

```diff
@@ -533,6 +537,36 @@ Note that the "all in one" node must be the "master". openshift-ansible
 expects at least one node in the `masters` Ansible group.
 
 
+## Multi-env Deployment Configuration
+
+If you want to deploy multiple OpenShift environments in the same OpenStack
+project, you can do so with a few configuration changes.
+
+First, set the `openshift_openstack_clusterid` and `openshift_openstack_stack_name`
+options in `inventory/group_vars/all.yml` file with specific unique names for clusters.
+
+```
+vi inventory/group_vars/all.yml
+
+openshift_openstack_clusterid: foobar_clusterid
+openshift_openstack_stack_name: foobar_stack_name
```

Comment:
This should never be set by end users. It is calculated from the domain name and cluster ID. If anything, I'm in favour of renaming `openshift_openstack_stack_name` to `l_openshift_openstack_stack_name` to make it clearer (but that should be done in a separate patch).